src/middlewares/errorHandler.ts

```diff
@@ -5,14 +5,14 @@ import logger from '../lib/logger'
 export default function errorHandler (error: BaseError, req: Request, res: Response, _next: NextFunction): void {
   const status = error.status || 500
   const message = error.message || 'Something went wrong '
-  const meta = error.meta || {}
+  const validationErrors = error.meta || {}
```

Comment:
You don't need to change anything in this file, I think. That is general meta, not specifically validationErrors. Instead you need to have correct key passed in this meta. (validationErrors in your case)